config/locales/cy/en.yml

```diff
@@ -0,0 +1,49 @@
+cy:
+  date:
+    abbr_day_names:
+      - Sul
+      - Llun
+      - Maw
+      - Mer
+      - Iau
+      - Gwe
+      - Sad
+    abbr_month_names:
```

Comment:
I think we'll need a line with an empty month here (for no 0 month) as per https://github.com/rails/rails/blob/main/activesupport/lib/active_support/locale/en.yml